### PR TITLE
PORTALS-2064: change layout of the header card

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Normal card rendering:
Screen Shot 2022-02-09 at 10 20 19 AM
Same card rendered as a Header card:
Screen Shot 2022-02-09 at 10 21 09 AM

File: src/lib/containers/HeaderCard.tsx [hunk 1]
```diff
@@ -68,49 +68,58 @@ const HeaderCard: React.FunctionComponent<HeaderCardProps> = ({
...
-                secondaryLabelLimit={secondaryLabelLimit}
-                values={values}
-              />
-            )}
+      <div className="container-fluid container-full-width">
+        <div className="row">
+          <div className="col-md-offset-1 col-md-10">
+            <div className="SRC-portalCardMain">
+              {icon}
+              <div style={{ width: '100%' }}>
```
Comment (nickgros) [inline src/lib/containers/HeaderCard.tsx:76]: Reason for inline styles? I have no preference whether they are CSS or inline if it's not reusable (inline makes more sense to me, honestly), but we haven't been doing this in the past.

File: src/lib/containers/HeaderCard.tsx [hunk 2]
```diff
@@ -68,49 +68,58 @@ const HeaderCard: React.FunctionComponent<HeaderCardProps> = ({
...
+                        <MarkdownSynapse markdown={description} />
+                      ) : (
+                        description
+                      )}
+                    </span>
+                  )}
+                </div>
+                <div
+                  style={{
+                    borderTop: '1px solid rgba(26, 28, 41, 0.2)',
```
Comment (nickgros) [inline src/lib/containers/HeaderCard.tsx:103]: Even if intentionally using inline styles, not sure if color should be inline, but maybe this is a reasonable special case

File: src/lib/containers/HeaderCard.tsx [hunk 3]
```diff
@@ -68,49 +68,58 @@ const HeaderCard: React.FunctionComponent<HeaderCardProps> = ({
...
+                <div className="SRC-cardContent" style={{ marginLeft: '15px' }}>
+                  <div className="SRC-type">{type}</div>
+                  <div>
+                    <h3 className="SRC-boldText" style={{ margin: 'none' }}>
+                      {href ? (
+                        <a target={target} href={href} className="highlight-link">
+                          {title}
+                        </a>
+                      ) : (
+                        <span> {title} </span>
```
Comment (nickgros) [inline src/lib/containers/HeaderCard.tsx:86]: Should any of these use `<Typography>`?

Comment (jay-hodgson) [reply src/lib/containers/HeaderCard.tsx:86]: Yes, did not update in this PR.

File: src/lib/containers/HeaderCard.tsx (hunk 2 above)
Comment (jay-hodgson) [reply src/lib/containers/HeaderCard.tsx:103]: Agreed that color should not be inline, did not update in this PR.

**Review by nickgros — APPROVED** [review]

**Pull request merged**